### PR TITLE
adding bug_report.yml file

### PR DESCRIPTION
**Pull request by hemantsoni42**
This is a DEMO file.

### CODE REVIEW

**Comment by hemantsoni42** [comment]
#14

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: "Logs"
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: bash
+    validations:
+      required: false
+  - type: dropdown
```
Comment (drkennetz) [inline .github/ISSUE_TEMPLATE/bug_report.yml:55]: I think we can remove the `browsers` dropdown, lines 55-68.

File: .github/ISSUE_TEMPLATE/feature_report.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: "💡 Feature Request"
+description: Create a new ticket for a new feature request
+title: "[REQUEST] - <title>"
+labels: [
+  "question"
+]
+body:
+  - type: input
```
Comment (drkennetz) [inline .github/ISSUE_TEMPLATE/feature_report.yml:8]: can remove lines 8-31. I want this to include a feature someone may want in the future but hasn't necessarily implemented. 

I'm okay with keeping the sections:
1. Summary
2. Basic Example
3. Drawbacks

Comment (drkennetz) [reply .github/ISSUE_TEMPLATE/feature_report.yml:8]: Let me know if you have questions.

**Review by drkennetz — CHANGES_REQUESTED** [review]

**Comment by hemantsoni42** [comment]
Closes #14

**Review by drkennetz — APPROVED** [review]
Thanks for working through this!

**Pull request merged**